sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/application/tasks/CreateNewTaskSteps.java

```diff
@@ -0,0 +1,169 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2021 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.sormas.e2etests.steps.application.tasks;
+
+import static org.sormas.e2etests.pages.application.tasks.CreateNewTaskPage.*;
+
+import com.google.common.truth.Truth;
+import cucumber.api.java8.En;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import javax.inject.Inject;
+import org.sormas.e2etests.helpers.WebDriverHelpers;
+import org.sormas.e2etests.pojo.Task;
+import org.sormas.e2etests.services.TaskService;
+
+public class CreateNewTaskSteps implements En {
+  public static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("M/dd/yyyy");
+  public static final DateTimeFormatter TIME_FORMATTER = DateTimeFormatter.ofPattern("HH:mm");
+  protected static Task task;
+  private final WebDriverHelpers webDriverHelpers;
+
+  @Inject
+  public CreateNewTaskSteps(WebDriverHelpers webDriverHelpers, TaskService taskService) {
+    this.webDriverHelpers = webDriverHelpers;
+
+    When(
+        "^I create a new task with specific data$",
```

Comment:
I usually parse data via tables in steps 
https://www.baeldung.com/cucumber-data-tables
but I'm assuming that this will be for tomorrow automation presenation and it's using a more general approach than cucumber.
so it's just a note. everything looks fine.

Reply:
Regarding this, the data is generate in the task service, we want to be sure the creation of the task with all fields is possible